curriculum/challenges/english/10-coding-interview-prep/rosetta-code/soundex.md

```diff
@@ -8,7 +8,17 @@ dashedName: soundex
 
 # --description--
 
-Soundex is an algorithm for creating indices for words based on their pronunciation. The goal is for homophones to be encoded to the same representation so that they can be matched despite minor differences in spelling (from [the WP article](https://en.wikipedia.org/wiki/soundex)). There is a major issue in many of the implementations concerning the separation of two consonants that have the same soundex code! According to the [official Rules](https://www.archives.gov/research/census/soundex.html). So check for instance if **Ashcraft** is coded to **A-261**.
+**Soundex Algorithm** deals with the *intentions* of the words. It creates a representation for similar sounding words. 
+
+It is mainly used in searching for <em>names</em> and <em>addresses</em>, because the person, who inputting the <em>names</em>, can focus on how they sound instead of the correct spelling of the <em>names</em>.
```

Comment:
```suggestion
It is used for searching <em>names</em> and <em>addresses</em>. This means that the person who filled in the <em>name</em>, can focus on how it sounds instead of correcting the spelling of <em>names</em>.
```